tasks/main.yml

```diff
@@ -32,24 +32,24 @@
   sudo: yes
   sudo_user: "{{ server_user }}"
 
-- name: unpack ruby2.3.3
+- name: unpack new ruby version
   unarchive:
     src: "{{ ruby_version_download_path }}"
-    dest: /home/vagrant
+    dest: "{{home_path}}"
   sudo: yes
   sudo_user: "{{ server_user }}"
 
 - name: install new ruby version
   command: ./configure
   args:
-    chdir: /home/vagrant/ruby-"{{ruby_version}}"/
+    chdir: "/home/vagrant/ruby-{{ruby_version}}/"
```

Comment:
Let's avoid the last `/` . we can assume with chdir that we refer to a directory